src/ui/components/App.tsx

```diff
@@ -79,7 +81,10 @@ function App({ theme, recordingId, modal, updateNarrowMode }: AppProps) {
   const auth = useAuth0();
   const { claims } = useToken();
 
-  setTelemetryContext(claims?.hasura.userId, auth.user?.email);
+  const userId = claims?.hasura.userId;
+  const { isInternal, email } = useGetUserInfo();
+
+  setTelemetryContext(userId, email);
```

Comment:
I wonder if we should have`setTelemetryContext` here or next to  `createSession`

Reply:
It _is_ a bit confusing but they're separate things. Set telemetry context doesn't determine whether the telemtry gets sent, only what context is attached.